tasks/frontEnd/src/components/ReviewForm.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <Form @submit="handleSubmit">
+    <TextField label="Shop name" v-model="review.shopName" autoComplete="off" required :error="shopNameError"></TextField>
+    <TextField label="Product name" v-model="review.productName" autoComplete="off" :error="productNameError"></TextField>
+    <TextField label="Your name" v-model="review.userName" autoComplete="off" :error="userNameError"></TextField>
+    <TextField type="email" label="Your email" v-model="review.userEmail" autoComplete="email" :error="userEmailError"></TextField>
+    <Text>Rating</Text>
+    <InlineStack gap="200">
+      <RadioButton v-for="n in 5" :label="n" :value="n" v-model="review.rating" :checked="review.rating == n" />
+    </InlineStack>
+    <TextField label="Review" v-model="review.reviewBody" :multiline="4" autoComplete="off" :error="reviewBodyError" />
+    <Checkbox label="Verified review" v-model="review.verified" />
+    <DropZone accept="image/*" type="image" @drop="handleDrop" label="Related images">
+      <Layout v-if="review.files.length" vertical>
+        <LayoutSection v-for="file, index in review.files" :key="index">
```

Comment:
Nit: Usually we avoid setting `key` to index in vue. Do you know why?

Reply:
Yes, using an index can cause issues when reordering lists. I didn't think this would be an issue for this use-case but see the benefit of changing for the sake of scalability